src/app/components/table/table.ts

```diff
@@ -5788,7 +5788,7 @@ export class ColumnFilter implements AfterContentInit {
             const documentTarget: any = this.el ? this.el.nativeElement.ownerDocument : 'document';
 
             this.documentClickListener = this.renderer.listen(documentTarget, 'mousedown', (event) => {
-                const dialogElements = document.querySelectorAll('[role="dialog"]');
+                const dialogElements = document.querySelectorAll('[role="filterDialog"]');
```

Comment:
You can try a different approach instead of adding and checking a random role. I'm not sure but, checking the elements parent would work.

Reply:
Yes, it is more appropriate to do so, I will do as described here.